projects/libraw/build.sh

```diff
@@ -15,12 +15,30 @@
 #
 ################################################################################
 
+# copy corpuses
+cp $SRC/libraw_cr2_fuzzer_seed_corpus.zip \
+    $SRC/libraw_nef_fuzzer_seed_corpus.zip \
+    $SRC/libraw_raf_fuzzer_seed_corpus.zip \
+    $OUT/
+
 # build project
-./mkdist.sh
+autoreconf --install
 ./configure --disable-examples
 make
 
 # build fuzzers
 $CXX $CXXFLAGS -std=c++11 -Ilibraw \
     $SRC/libraw_fuzzer.cc -o $OUT/libraw_fuzzer \
     $LIB_FUZZING_ENGINE lib/.libs/libraw.a
+
+$CXX $CXXFLAGS -std=c++11 -Ilibraw \
+    $SRC/libraw_fuzzer.cc -o $OUT/libraw_cr2_fuzzer \
```

Comment:
I brought this up during Kostya fuzzing workshop. Each of these corpuses are entirely different RAW file format and as such access different code paths. It seems the general advice was to have a fuzzer for each corpus to ensure better coverage?

Reply:
underneath it can trigger three different parsing paths, but your fuzzer is still calling through main libraw processing path. so, i dont see any usefulness of seperating into three fuzzers. if you want to do this properly, then you should calling the different parsing api for these formats in the individual fuzzers. 
@kcc as well
can we ask your host to review this cl as well.

Reply:
LibRaw has one parsing API (LibRaw::unpack) for all formats, the code heavily diverges based on the file format found in the header. If one fuzzer with multiple corpuses can handle this, I can make the change. @kcc what do you recommend as well?

(@nchusid as well :) )

Reply:
I (Jamie's host) am not very familiar with oss-fuzz or fuzzing best practices, but I do see some sense in his strategy of different fuzzers for each format to encourage better fuzzing coverage, since the code paths invoked for each format can be completely different.

Would separate fuzzers improve the per-format input generation? If so, I would think multiple fuzzers to be better.

Reply:
lets give this a shot!. go each format have a lot of its own sample corpus files ? more samples will help, can be added in another pr.

Reply:
Yea! I started with the top 3 formats, to see how much of an improvement there is. Can add more afterwards!

Reply:
+1 for trying to call the underlying API that deals with one specific file format at a time.